### ISSUE TITLE
Latex not running in Linux (flatpack installation)

### ISSUE THREAD

**Issue by lostinafro** [otfried/ipe#479]
Hi,

I start saying that this software is always been useful to me and I thank the developers for it. I always used the binary version installed on Ubuntu, but I liked the idea of employing the flatpak version: easier to install and always up to date.

However, using the flatpak installation, latex does not run anymore.
![Screenshot from 2023-09-18 14-53-41](https://github.com/otfried/ipe/assets/51850828/9068caf0-e02c-4af4-bee0-c61714958468)

Even when I add to the environmental variable `IPELATEXPATH=/usr/bin/` the same error message appears:
![Screenshot from 2023-09-18 14-57-22](https://github.com/otfried/ipe/assets/51850828/203dc0d1-ea70-4e7e-835a-348a85f16214)
Note that IPE is not able to find `/usr/bin/pdflatex`, but the latex programs are actually in `/usr/bin/`, as shown here
![Screenshot from 2023-09-18 15-15-42](https://github.com/otfried/ipe/assets/51850828/cd478f40-5313-4920-956d-54805477142b)
Remark the errors related to the QStandardPaths permissions, as you can inall the screen at startup of IPE (I don't know if these are related to the problem).

I tried give system-wide permissions to the flatpak application as well as creating a soft link of pdflatex, luatex, and xelatex in the HOME directory, but the same problems occurs: `sh: line 1: /home/MYPC/.ipe/pdflatex: No such file or directory`

Is a problem related to my settings? 
Thank you

**Comment by otfried** [comment]
It is simply not possible to use a local installation of latex (in your `/usr/bin` directory) from a flatpak.

This is a design decision of flatpak: to protect your system, the software in the flatpak has access to only those parts of the file system that you give it access to.  The flatpak itself is a complete file system image, that already contains its own `/usr/bin`, and it is not possible to mix the one from your distribution with the one inside the flatpak.

There are three options you can use to run Latex from Ipe in a flatpak, described here: https://github.com/flathub/org.otfried.Ipe

In short, if you don't want to use online compilation, you would have to install the flatpak for Latex, or instead install latex yourself in a separate directory like `/opt/texlive` and give the flatpak access to this.
